packages/quantic/force-app/main/default/lwc/quanticSearchBoxInput/quanticSearchBoxInput.js

```diff
@@ -240,6 +242,14 @@ export default class QuanticSearchBoxInput extends LightningElement {
     this.adjustTextAreaHeight();
   }
 
+  onClearButtonFocus() {
+    this.inputClearButton.setAttribute('is-focused', 'true');
+  }
+
+  onClearButtonBlur() {
```

Comment:
this method prevents the issue where the focus is still on the `inputClearButton` and where:
```
tab on clear Icon
shift + tab back on input
press ENTER 
```

was no longer triggering a search

Reply:
I don't see how manually controlling the focus is the solution to the issue...

You have the problem of the ENTER no longer triggering a search because you're manually checking this:

```
} else if (this.inputClearButton?.getAttribute('is-focused') === 'true') {
      this.clearInput();
```